ctapipe/ml/tools/apply.py

```diff
@@ -186,6 +243,110 @@ def _apply(self, reconstructor, parameter):
 
         return vstack(tables)
 
+    def _apply_disp(self):
+        # Different prefix? Two algorithms -> How to combine them?
```

Comment:
Yes, different prefix (see my other comment). Applying disp makes only sense if we apply both reconstructors.